src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemDeprecationLabel.xaml

```diff
@@ -0,0 +1,71 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.PackageItemDeprecationLabel"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:sys="clr-namespace:System;assembly=mscorlib"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="300">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <nuget:SharedResources />
+      </ResourceDictionary.MergedDictionaries>
+
+      <nuget:DeprecationToDeprecationLabelStateConverter x:Key="DataToStateConverter" />
+      <nuget:FormatedStringPartConverter x:Key="FormatedStringPartConverter" />
+      <sys:Int32 x:Key="ParamPrefix">0</sys:Int32>
+      <sys:Int32 x:Key="ParamSuffix">2</sys:Int32>
+
+      <Style x:Key="StyleLink" TargetType="TextBlock">
```

Comment:
The name of this style implies it's for a link but it's used for the Deprecated, no Alternative case too which does not have a link. Consider renaming to reflect its purpose / style. BoldTextStyle, as an example.

Reply:
Renamed to `DeprecationTextBlockStyle`